src/router/index.ts

```diff
@@ -40,12 +40,13 @@ export default route(function (/* { store, ssrContext } */) {
     // quasar.conf.js -> build -> publicPath
     history: createHistory(process.env.VUE_ROUTER_BASE),
   });
-
+  const enabled = false;
```

Comment:
Why we need static constant inside if condition?

Reply:
@tmszi this was changed for testing purposes. I accidentally commited it, but in current version it should be cleaned up and removed.